frontend/public/components/masthead.tsx

```diff
@@ -9,12 +10,96 @@ import * as dedicatedLogoImg from '../imgs/openshift-dedicated-logo.svg';
 import { FLAGS, connectToFlags, flagPending } from '../features';
 import { authSvc } from '../module/auth';
 import { Dropdown, ActionsMenu } from './utils';
+import { SoftwareDetails } from './software-details';
 
 import { coFetchJSON } from '../co-fetch';
 import { SafetyFirst } from './safety-first';
 
 const developerConsoleURL = (window as any).SERVER_FLAGS.developerConsoleURL;
 
+export class HelpMenu extends React.Component<HelpMenuProps, HelpMenuState> {
+  constructor (props) {
+    super(props);
+    this.state = { showModal: false };
+    this.close = this.close.bind(this);
+    this.open = this.open.bind(this);
+    this.redirect = this.redirect.bind(this);
+  }
+
+  close() {
+    this.setState({ showModal: false });
+  }
+
+  open() {
+    this.setState({ showModal: true });
+  }
+
+  redirect() {
+    window.open('https://docs.openshift.com/online/welcome/index.html', '_blank');
+  }
+
+  render() {
+    const {username} = this.props;
+    return <React.Fragment>
+      <ActionsMenu
+        actions={[
+          {label: 'Documentation', callback: this.redirect},
+          {label: 'About', callback: this.open}]}
+        noButton
+        noCaret
+        title={<i className="fa fa-question-circle-o co-masthead__help-icon" />} />
+      <AboutModal className="co-masthead__modal" logo={okdLogoImg} altLogo="OKD" productTitle="OpenShift" show={this.state.showModal} onHide={this.close}>
```

Comment:
We're not always showing the OKD logo. We need to show the right branding depending on what's running (OKD, OCP, Online, Dedicated)